control/plugin/processor_test.go

```diff
@@ -81,7 +81,7 @@ func (s *MockProcessorSessionState) Token() string {
 	return s.token
 }
 
-func (m *MockProcessorSessionState) ResetHeartbeat() {
+func (s *MockProcessorSessionState) ResetHeartbeat() {
```

Comment:
Since your other mock examples started with m this should be ms so it's clear it's a mock.